### ISSUE TITLE
./X auth:basic:delete fails to remove token from collection

### ISSUE THREAD

**Issue by RoboSparrow** [Brightcookie/lxHive#131]
basicTokens
```bash
./X auth:basic:list 
[
    {
        "_id": {
            "$id": "5919145ccaa9a74a108b4567"
        },
        "userId": {
            "$id": "5919135bcaa9a7de0e8b4567"
        },
        "key": "<removed>",
        "secret": "<removed>",
        "expiresAt": null,
        "createdAt": {
            "sec": 1494815836,
            "usec": 0
        },
        "name": "client1",
        "description": "",
        "scopeIds": [
            {
                "$id": "59190908caa9a7450d8b4570"
            }
        ]
    },
    {
        "_id": {
            "$id": "591a465bcaa9a784678b4567"
        },
        "userId": {
            "$id": "591a459fcaa9a738678b4567"
        },
        "key": "<removed>",
        "secret": "<removed>",
        "expiresAt": null,
        "createdAt": {
            "sec": 1494894171,
            "usec": 0
        },
        "name": "testing",
        "description": "testing",
        "scopeIds": [
            {
                "$id": "59190908caa9a7450d8b4570"
            }
        ]
    }
]
```

trying to delete `clientt1` token

```bash
./X auth:basic:delete
Please enter the the client ID of the token you wish to delete: 5919135bcaa9a7de0e8b4567
Are you sure (y/n): y
Supertoken successfully deleted!

```
but it is still there
```bash
./X auth:basic:list 
[
    {
        "_id": {
            "$id": "5919145ccaa9a74a108b4567"
        },
        "userId": {
            "$id": "5919135bcaa9a7de0e8b4567"
        },
        "key": "<removed>",
        "secret": "<removed>",
        "expiresAt": null,
        "createdAt": {
            "sec": 1494815836,
            "usec": 0
        },
        "name": "client1",
        "description": "",
        "scopeIds": [
            {
                "$id": "59190908caa9a7450d8b4570"
            }
        ]
    },
    {
        "_id": {
            "$id": "591a465bcaa9a784678b4567"
        },
        "userId": {
            "$id": "591a459fcaa9a738678b4567"
        },
        "key": "<removed>",
        "secret": "<removed>",
        "expiresAt": null,
        "createdAt": {
            "sec": 1494894171,
            "usec": 0
        },
        "name": "testing",
        "description": "testing",
        "scopeIds": [
            {
                "$id": "59190908caa9a7450d8b4570"
            }
        ]
    }
]
```

**Comment by sraka1** [comment]
Should be fixed in #134 - user needs to provide key (HTTP Basic Auth username) of the key they want to delete.

**Comment by RoboSparrow** [comment]
resolved in https://github.com/Brightcookie/lxHive/pull/134
